### DISCUSSION TITLE
Would it be possible to add a custom tool in the Painting default menu?

### DISCUSSION THREAD

**Discussion by saif-ellafi** [hm21/pro_image_editor#269] [Q&A]
Hi,

Would it be possible to modify the original painting menu without rewriting it all?
I would like to add my custom Text tool inside the painting menu! (Along the circle, line, etc.)

Best regards

**Comment by hm21** [comment]
You can open the painting editor by yourself by replacing the bottomBar with `customWidgets` in the `configs`. This approach allows you to add `TextLayers` within the `initConfigs` which will be rendered inside the painting editor.
